src/sagemaker_inference/transformer.py

```diff
@@ -48,6 +48,7 @@ def find_spec(module_name):
 from sagemaker_inference.default_inference_handler import DefaultInferenceHandler
 from sagemaker_inference.errors import BaseInferenceToolkitError, GenericInferenceToolkitError
 
+import inspect
```

Comment:
Import statements are in the wrong order. 'import inspect' should be before 'from sagemaker_inference.errors import BaseInferenceToolkitError, GenericInferenceToolkitError' and in a different group.

Reply:
Also expect 2 blank lines between imports and actual code